src/common/generator.js

```diff
@@ -0,0 +1,62 @@
+const faker = require('@faker-js/faker');
+const CATEGORY = ["expenses", "income"];
+
+const random = (array) => {
+    return array[Math.floor(Math.random() * array.length)];
+};
+
+class Generator {
```

Comment:
usually when creating mocked objects, the name should suggest something with Mock, so maybe MockGenerator, to understand what the generator is about